src/components/Form/Dropdown.js

```diff
@@ -179,6 +180,9 @@ let Dropdown = (props, ref) => {
 
     const debouncedSearchChange = (value) => {
         setSearchQuery(value);
+        if (typeof searchCallback === "function") {
+            searchCallback(value);
```

Comment:
lets name this function as per the React standards for an event callback `on<event>` , for example, `onSearchChange`